src/commands/synthetics/test.ts

```diff
@@ -35,12 +36,26 @@ export const getTestConfigs = async (
 
   const testConfigs = suites
     .map((suite) =>
-      suite.content.tests.map((test) => ({
-        // TODO SYNTH-12989: Clean up deprecated `config` in favor of `testOverrides`
-        testOverrides: replaceConfigWithTestOverrides(test.config, test.testOverrides),
-        id: normalizePublicId(test.id) ?? '',
-        suite: suite.name,
-      }))
+      suite.content.tests.map((test) => {
+        const common = {
+          // TODO SYNTH-12989: Clean up deprecated `config` in favor of `testOverrides`
+          testOverrides: replaceConfigWithTestOverrides(test.config, test.testOverrides),
+          suite: suite.name,
+        }
+
+        if (isLocalTriggerConfig(test)) {
+          return {
+            ...common,
+            id: test.id && (normalizePublicId(test.id) ?? ''),
+            localTestDefinition: test.localTestDefinition,
+          }
+        }
+
+        return {
+          ...common,
+          id: normalizePublicId(test.id) ?? '',
+        }
+      })
```

Comment:
Following that an ltd trigger might not need an id:
```suggestion
        return {
          ...common,
          ...(
            isLocalTriggerConfig(test)
              ? { localTestDefinition: test.localTestDefinition }
              : { id: normalizePublicId(test.id) }
          )
        }
```